### ISSUE TITLE
Submenus in the application menu are not opened each time

### ISSUE THREAD

**Issue by nicwe** [fluentribbon/Fluent.Ribbon#698]
If you use an application menu with submenus, some submenus will not open each time you move the mouse to the parent menuitem.

Steps to reproduce:

1. Start the showcase application
2. Switch to the application menu (not backstage)
3. Open the application menu
4. Move the mouse over the "File" menu to the "Save As" menu (the submenu will open)
5. Move the mouse over the opened submenu to a free position in the window (not ribbon menu)
6. Click with the mouse to close the application menu
7. Open the application menu again
8. Move the mouse over the "File" menu  to the "Save As" menu agein (the submenu will NOT open)

- If you repeat the steps you will see that every second time the submenu will not be opened.
- In case the submenu will not opened, you can hover over some other menu items, then it will work.
- If you move the mouse not over the "File" menu to the "Save As" menu (maybe over the "Recent Files") the submenu will opened each time.

I think it is related to #510 

Environment

- Fluent.Ribbon v6.1.0
- Windows 10
- .NET Framework 4.5

**Comment by batzen** [comment]
As far as i remember i fixed this in version 7.0. could you try to reproduce this using the preview of version 7.0? I am currently on vacation and can't try this myself.

**Comment by nicwe** [comment]
To test this I followed the basic instructions from here "https://fluentribbon.github.io/documentation/basic-setup". I replaced the <Fluent:Backstage> element with the following code:

```
<Fluent:ApplicationMenu>
  <Fluent:MenuItem Header="1" />
  <Fluent:MenuItem Header="5" >
    <Fluent:MenuItem Header="5.1"/>
  </Fluent:MenuItem>
  <Fluent:MenuItem Header="8"/>
  <Fluent:MenuItem Header="9">
    <Fluent:MenuItem Header="9.1"/>
  </Fluent:MenuItem>    
</Fluent:ApplicationMenu>
```
But the problem still exists. I figuered out that the problem exists only when you leave the application menu (step 5 see above) to the right. If you leave every time the application menu in direction down the problem is not present.

I wish you a nice holiday :-)

---

Environment

- Fluent.Ribbon v7.0.0-alpha0586
- Windows 10
- .NET Framework 4.5
